roles/ceph-infra/tasks/tune_kernel_parameters.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "tune kernel parameters {{ tune_kernel_parameters }} and merge with existing templates from /etc/grub.d"
+  command: >
+    grubby
+    --update-kernel=ALL
+    --copy-default
+    --args="{{ tune_kernel_parameters | quote }}"
+  changed_when: false
+
+- name: reboot the system to apply new kernel parameters
```

Comment:
We might need to serialize this task.

Reply:
This will probably also need to use the changed state from the previous task if we want to avoid to reboot the node on every playbook call